lxd/cluster/membership.go

```diff
@@ -116,18 +116,20 @@ func Bootstrap(state *state.State, gateway *Gateway, name string) error {
 		return err
 	}
 
-	// The cluster certificates are symlinks against the regular node
-	// certificate.
-	for _, ext := range []string{".crt", ".key", ".ca"} {
-		if ext == ".ca" && !shared.PathExists(filepath.Join(state.OS.VarDir, "server.ca")) {
-			continue
-		}
-		err := os.Symlink("server"+ext, filepath.Join(state.OS.VarDir, "cluster"+ext))
+	// The cluster CA certificate is a symlink against the regular server CA certificate.
+	if shared.PathExists(filepath.Join(state.OS.VarDir, "server.ca")) {
```

Comment:
@stgraber I wasn't sure about this bit. If there is a CA cert available, should the newly generated cluster certificate be signed by that CA cert?

Reply:
No, we don't have a key for .ca, it's just a certificate used to check client connections.